Engines/Wine/Verbs/VK9/script.js

```diff
@@ -36,25 +33,35 @@ class VK9 {
         const system32directory = this.wine.system32directory();
         const system64directory = this.wine.system64directory();
 
-        if (operatingSystemFetcher.fetchCurrentOperationSystem() != "Linux") {
-            wizard.message(
-                tr(
-                    "VK9 might not work correctly on macOS. This is depending on Metal api support and MoltenVK compatibility layer advancement"
-                )
-            );
-        } else {
-            wizard.message(
-                tr(
-                    "Please ensure you have the latest drivers (418.30 minimum for NVIDIA and mesa 19 for AMD) or else VK9 might not work correctly."
-                )
-            );
+        if (operatingSystemFetcher.fetchCurrentOperationSystem().getFullName() !== "Linux")
+        {
+            const question = tr("VK9 is currently unsupported on non-Linux operating systems due to MoltenVK implementation being incomplete. Select how do you want to approach this situation.")
+            const choices = [
+                tr("YES, continue with VK9 installation regardless"),
+                tr("NO, quit script alltogether"),
+                tr("Exit VK9 Installer, but continue with the script")
+            ];
+
+            const answer = wizard.menu(question, choices);
+
+            switch (answer.index) {
+                case 1:
+                    // choice: "NO, quit script alltogether"
+                    throw new Error("User aborted the script.");
+                case 2:
+                    // choice: "Exit VK9 Installer, but continue with the script"
+                    return this;
+                default:
```

Comment:
Alternatively to `default:` we could make the `YES, continue with VK9 installation regardless` choice explicit by using:

```javascript
case 0:
   // choice: "YES, continue with VK9 installation regardless"
   break;
```